File: dev/requirements/screen/cases/set-list-grid.4.1.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 4.1 — nine cards in three columns, filling the height the joke panel leaves.
let setListGrid_4_1 = RequirementCase(id: "4.1") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "set-list-grid", id: "4.1", region: .grid)
    }
}
#endif

File: dev/requirements/screen/cases/set-list-grid.4.1.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 4.1 — The set list renders as a grid of cards, each carrying its joke's title,
/// filling the height the joke panel leaves.
final setListGrid_4_1 = RequirementCase('4.1', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.grid,
    slug: 'set-list-grid',
    id: '4.1',
  );
});
